lib/cog/commands/pluck.ex

```diff
@@ -0,0 +1,115 @@
+defmodule Cog.Commands.Pluck do
+  use Spanner.GenCommand.Base, bundle: Cog.embedded_bundle, enforcing: false, calling_convention: :all
+  alias Cog.Commands.Helpers, as: CommandHelpers
+
+  @moduledoc """
+  Returns the specified fields from a set of inputs.
+  Be sure to add quotes when traversing a JSON path.
+
+  ## Example
+
+      @bot #{Cog.embedded_bundle}:rules --list --for-command="#{Cog.embedded_bundle}:permissions" | #{Cog.embedded_bundle}:pluck --fields=id
```

Comment:
I know this might not be as future proof, but the `Cog.embedded_bundle` bit makes it a little bit harder to read in the raw docs.